spidermon/contrib/scrapy/pipelines.py

```diff
@@ -56,6 +56,7 @@ def set_validators(loader, schema):
         for loader, name in [
             (cls._load_jsonschema_validator, "SPIDERMON_VALIDATION_SCHEMAS"),
             (cls._load_schematics_validator, "SPIDERMON_VALIDATION_MODELS"),
+            (cls._load_cerberus_validator, "SPIDERMON_VALIDATION_CERBERUS")
```

Comment:
Related to my previous comment about the documentation, including a documentation about `SPIDERMON_VALIDATION_CERBERUS` is more suitable than the change you did related to JSON Schema.

Reply:
Roger that, will be implemented.